src/core/issues-notes.js

```diff
@@ -143,7 +143,7 @@ export async function run(conf, doc, cb) {
     if (conf.githubAPI) {
       try {
         const json = await ghFetch(conf.githubAPI);
-        const issueUrl = issueBase || json.html_url + "/issues/";
+        var issueUrl = issueBase || json.html_url + "/issues/";
```

Comment:
Can you explain why var here?

Reply:
As const is blocked scoped and I declared it in try, I couldn't use the same variable in finally so had  to make it var. 
Is there another turn around?

Reply:
I'll have a look, but in principle, if we need a `var` then it's likely there is a bug in the design somewhere.

Reply:
I can make another wrapper function which would take run function as argument and return run.catch() and we can handle errors there then. 
What do you say @marcosscaceres ?

Reply:
we can seperate it in another functinon like before and then use .catch on it ? @marcoscaceres

Reply:
No, that would not be ideal either. This module is not doing anything special that would warrant it having a different design than any other module.

Reply:
re-assigning the previous global variable is also not an option, right @marcoscaceres